mobile/src/packages/meditation/libs/types/types.ts

```diff
@@ -0,0 +1,7 @@
+export {
+  type MeditationEntryCreateForm,
+  type MeditationEntryCreateRequestDto,
+  type MeditationEntryCreateResponseDto,
```

Comment:
The same, we don't need create dtos here for now

Reply:
@Dmytro-Kucherenko they are needed because this type is used in my task component
![Снимок экрана 2023-09-22 155629](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/111873795/69d4bee9-224e-4676-9df1-afcb1cff0cc5)